.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!--
+Check the Rancher docs issues to see if there is an existing issue for this pull request. If there is, enter the issue number below.
+-->
+
+Fixes #[issue_number]
+
+## Reminder
+
+<!--
+Verify if changes pertain to other versions of Rancher. If they do, finalize the edits on one version of the page, then apply the edits to the other versions.
```

Comment:
```suggestion
<!-- Reminder
Verify if changes pertain to other versions of Rancher. If they do, finalize the edits on one version of the page, then apply the edits to the other versions.
```

I think we should make the `Reminder` label part of the comment. In the other instances where a header is used we're expecting the PR creator to fill in some information in the template, but that's not case here.